src/chunk.rs

```diff
@@ -0,0 +1,105 @@
+use std::iter::*;
+use std::fmt::Debug;
+
+/// Chunk of code text
+#[derive(Debug)]
+pub enum Chunk {
+    Text(String),
+    EndLine,
+}
```

Comment:
If the `Chunk`'s purpose is to deal with newlines, consider a wrapper around `File` that would just do `s/\n/\r\n/g`, should be simpler.